### ISSUE TITLE
Better communication from discord bridge

### ISSUE THREAD

**Issue by ChillerDragon** [jxsl13/TeeworldsEconDiscordModerationBot#14]
The ?say command can be confused with the info messages on the server. Not related to this repo but maybe prefixing info messages with "[INFO]" would be nice. And maybe add a alternative or edit say to prefix "[DISCORD]\<chattername\>" so you know who is talking :)

**Comment by jxsl13** [comment]
I had the idea to optionally prefix it with the discord user names.

**Comment by jxsl13** [comment]
do you mean the announcements?

**Comment by jxsl13** [comment]
in regard to the info messages, do you mean the announcements?

**Comment by ChillerDragon** [comment]
> in regard to the info messages, do you mean the announcements?

Yes as far as I remember they also lack a prefix and could be an message by staff as well.

The things like "Did you know about @mods" :D (omg i pinged him again sry)
